openslides/users/rest_auth.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework.authentication import BaseAuthentication
+
+from openslides.config.api import config
+
+
+# TODO: Move this into the file 'auth' after fixing circular imports
```

Comment:
I think it is because you import the whole rest_api in utils/rest_api.py. The config is imported when the rest_api is initialized. So the file where the Authentication lays is loaded. If this file imports utils/rest_api.py, then there is the circle.